core/src/components/text-area/text-area.tsx

```diff
@@ -28,16 +28,31 @@ const onChange = (props: TextAreaProps) => (e: ChangeEvent) => {
 
 export interface TextAreaProps {
 	// Uncontrolled
+	/**
+	 * Initial value of the input
+	 */
 	defaultValue?: string;
 	forwardedRef?: ForwardedRef<HTMLTextAreaElement>;
 	// Controlled
 	value?: string;
+
+	/**
+	 * Handler to set the value
+	 */
 	setValue?: (value: string) => void;
```

Comment:
Both "value" and "setValue" should have description, as they are a pair and is not the same as (value, onChange)